components/molecules/QuestionAnswer/QuestionAnswer.js

```diff
@@ -83,7 +87,7 @@ QuestionAnswer.propTypes = {
   /** Additional style properties to apply to the container. */
   style: PropTypes.objectOf(PropTypes.string),
   /** The header of the QuestionAnswer section. */
-  header: PropTypes.string.isRequired,
+  header: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
```

Comment:
```suggestion
  header: PropTypes.node.isRequired,
```
Don't think you actually need to specify both string and node